src/Uno.UWP/Globalization/Language.Android.cs

```diff
@@ -0,0 +1,24 @@
+﻿#if __ANDROID__
+using Android.Content;
+using Android.Views.InputMethods;
+using Java.Interop;
+using Uno.UI;
+
+namespace Windows.Globalization
+{
+	public partial class Language
+	{
+		public static string CurrentInputMethodLanguageTag
+		{
+			get
+			{
+				var inputMethodManager = ContextHelper.Current.GetSystemService(Context.InputMethodService).JavaCast<InputMethodManager>();
+				var currentInputMethod = inputMethodManager.CurrentInputMethodSubtype;
+				return currentInputMethod.LanguageTag;
+			}
+		}
+
+		public static bool TrySetInputMethodLanguageTag(string languageTag) => false;
```

Comment:
We may want to log that it is not supported.

Reply:
The `false` should actually be ok - according to docs - https://docs.microsoft.com/en-us/uwp/api/windows.globalization.language.trysetinputmethodlanguagetag - it is expected that the method does nothing on all devices except IoT.

Reply:
That makes sense, thanks.